services/QuillLMS/client/app/bundles/Evidence/components/studentView/container.tsx

```diff
@@ -481,13 +481,14 @@ export const StudentViewContainer = ({ dispatch, session, isTurk, location, acti
       const innerElements = node.children.map((n, i) => convertNodeToElement(n, i, transformMarkTags))
       const stringifiedInnerElements = node.children.map(n => n.data ? n.data : n.children[0].data).join('')
       let className = ''
+      const key = `${Math.random()}`;
```

Comment:
Having a random key defeats the purpose of a key, because this will be regenerated on every rerender and then the key will change so React will have to rerender the components. I'd key it off the contents or maybe the index.